cmd/api/src/api/v2/auth/auth_test.go

```diff
@@ -3137,3 +3168,404 @@ func TestActivateMFA_Success(t *testing.T) {
 		require.Contains(t, rr.Body.String(), auth.MFAActivated)
 	}
 }
+
+// Can only replace a field value at the root of the JSON object
+// if field does not exist in jsonString, will effectively no-op
+func replaceFieldValueInJsonString(jsonString string, field string, value any) (string, error) {
+	var unmarshaled map[string]any
+	err := json.Unmarshal([]byte(jsonString), &unmarshaled)
+	if err != nil {
+		return "", err
+	}
+
+	if _, exists := unmarshaled[field]; exists {
+		unmarshaled[field] = value
+	}
+
+	modifiedJson, err := json.Marshal(unmarshaled)
+	if err != nil {
+		return "", err
+	}
+
+	return string(modifiedJson), nil
+}
+
+// Steel Thread Demo
```

Comment:
Was there a particular reason we chose to write this out as a very large test rather than utilizing the Matrix example?